scripts/quests/otherAreas/The_Sand_Charm.lua

```diff
@@ -0,0 +1,143 @@
+-----------------------------------
+-- The Sand Charm
+-----------------------------------
+-- Log ID: 4, Quest ID: 8
+-- !addquest 4 8
+-- Blandine  : !pos 23 -7 41 249
+-- Zexu      : !pos 31.511 -9.001 23.496 249
+-- Celestina : !pos -37.624 -16.050 75.681 249
+-----------------------------------
+
+local quest = Quest:new(xi.questLog.OTHER_AREAS, xi.quest.id.otherAreas.THE_SAND_CHARM)
+
+quest.reward =
+{
+    exp      = 2000,
+    gil      = 2000,
+    ki       = xi.ki.MAP_OF_BOSTAUNIEUX_OUBLIETTE,
+    fameArea = xi.fameArea.WINDURST,
+}
+
+quest.sections =
+{
+    {
+        check = function(player, status, vars)
+            return status == xi.questStatus.QUEST_AVAILABLE and
+                player:getFameLevel(xi.fameArea.WINDURST) >= 4 and
+                xi.settings.map.FISHING_ENABLE == true
+        end,
+
+        [xi.zone.MHAURA] =
+        {
+            ['Blandine'] =
+            {
+                onTrigger = function(player, npc)
+                    local xPos = player:getXPos()
+                    local zPos = player:getZPos()
```

Comment:
Could do with a comment here saying "this is for either side of the barrier, you should trigger on the north side, etc. etc. etc."